packages/cache/__tests__/saveCache.test.ts

```diff
@@ -220,65 +221,6 @@ test('save with reserve cache failure should fail', async () => {
   expect(getCompressionMock).toHaveBeenCalledTimes(1)
 })
 
-test('save with server error should fail', async () => {
```

Comment:
Restore these tests...

Reply:
Restore these tests, fix them if they are failing or explain why you removed them.